app/root.css

```diff
@@ -26,6 +26,25 @@
   /* dark theme at the end of file to overwrite all previous rules */
 }
 
+.loader {
+  border: 4px solid #f3f3f3; /* Light grey */
+  border-top: 4px solid #3498db; /* Blue */
```

Comment:
do both colors work well in both light and dark theme?

Reply:
<img width="1086" alt="image" src="https://user-images.githubusercontent.com/3942390/218086910-463c8a2b-662e-4d64-82de-7e544ca4b5ab.png">
Depends on what you want :) To be honest, this was pretty much the first css loader that I found, so any corrections/enhancements are welcome

Reply:
yeah, visibe enough for now, can be improved later if someone will find a better color